--- kyotocabinet-1.2.77/configure.in.orig	2018-10-28 18:41:50.000000000 +0100
+++ kyotocabinet-1.2.77/configure.in	2018-12-26 08:52:25.261569315 +0100
@@ -35,8 +35,8 @@
 MYPCFILES="kyotocabinet.pc"
 
 # Building flags
-MYCFLAGS="-Wall -ansi -pedantic -fPIC -fsigned-char -g0 -O2"
-MYCXXFLAGS="-Wall -fPIC -fsigned-char -g0 -O2"
+MYCFLAGS="-Wall -ansi -pedantic -fPIC -fsigned-char"
+MYCXXFLAGS="-Wall -fPIC -fsigned-char"
 MYCPPFLAGS="-I. -I\$(INCLUDEDIR) -I/usr/local/include"
 MYCPPFLAGS="$MYCPPFLAGS -DNDEBUG -D_GNU_SOURCE=1"
 MYCPPFLAGS="$MYCPPFLAGS -D_FILE_OFFSET_BITS=64 -D_REENTRANT -D__EXTENSIONS__"
@@ -202,28 +202,6 @@
 # Byte order
 AC_C_BIGENDIAN(MYCPPFLAGS="$MYCPPFLAGS -D_MYBIGEND")
 
-# System-depending optimization
-printf 'checking for 64-bit availability... '
-if printf 'main() {}' | $CC -xc -m64 -o config.tmp - >config.tmp 2>&1
-then
-  MYCFLAGS="-m64 $MYCFLAGS"
-  MYCXXFLAGS="-m64 $MYCXXFLAGS"
-  printf 'yes\n'
-else
-  printf 'no\n'
-fi
-if test "$enable_opt" != "no"
-then
-  printf 'checking for CPU optimization availability... '
-  if printf 'main() {}' | $CC -xc -march=native -o config.tmp - >config.tmp 2>&1
-  then
-    MYCFLAGS="-march=native $MYCFLAGS"
-    MYCXXFLAGS="-march=native $MYCXXFLAGS"
-    printf 'yes\n'
-  else
-    printf 'no\n'
-  fi
-fi
 printf 'checking for useless warnings... '
 if printf 'main() {}' | $CC -xc \
   -Wno-unused-but-set-variable -Wno-unused-but-set-parameter -o config.tmp - >config.tmp 2>&1
